tests/containers/secret.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright 2020-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: (podman|docker) secret
+# Summary: Test the `secret` subcommand for Docker and Podman
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use containers::common;
+use containers::container_images;
+
+sub run {
+    my ($self, $args) = @_;
+    my $runtime = $args->{runtime};
+    my $output = '';
+
+    select_serial_terminal();
+
+    # In Docker, secrets can only be used in a swarm
+    my $ip_addr = script_output("ip --brief -6 address show dev eth0 | tr -s ' ' | cut -d ' ' -f 3 | cut -d '/' -f 1");
+    record_info("docker swarm init", "Initializing docker swarm in IP $ip_addr");
+    script_run("$runtime swarm init --advertise-addr $ip_addr") if ($runtime =~ 'docker');
```

Comment:
if it is a pure docker command, I do not see the reason why to use $runtime.
Also, why `script_run`?